lib/puppet/environments.rb

```diff
@@ -4,7 +4,7 @@ module Puppet::Environments
   class EnvironmentNotFound < Puppet::Error
     def initialize(environment_name, original = nil)
       environmentpath = Puppet[:environmentpath]
-      super("Could not find a directory environment named '#{environment_name}' anywhere in the path: #{environmentpath}. Does the directory exist?", original)
+      super(_("Could not find a directory environment named '#{environment_name}' anywhere in the path: #{environmentpath}. Does the directory exist?", original))
```

Comment:
I think we want the first closing parens inside of `original` - that's not part of the string